astropy/coordinates/angle_formats.py

```diff
@@ -412,11 +413,15 @@ def degrees_to_dms(d):
 def dms_to_degrees(d, m, s=None):
     """
     Convert degrees, arcminute, arcsecond to a float degrees value.
-    """
-
+     """
     _check_minute_range(m)
     _check_second_range(s)
 
+    warn("dms_to_degrees (or creating an Angle with a tuple) has ambiguous "
```

Comment:
Why not use the `@deprecated` decorator?

Reply:
Hmm, at the time my motivation was because there's not a single clear "alternative" to use, but now that I look at how ``alternative`` is rendered I think I can make it work.  One sec.